src/app/modules/auth/classic-auth/classic-auth.service.ts

```diff
@@ -160,7 +165,37 @@ export class ClassicAuthService {
     }
   }
 
-  async activate (token: string) {
+  async resendActivationEmail (classicAuthActivateResendPayloadDto :ClassicAuthActivateResendPayloadDto, request: Request) {
+    const message = {
```

Comment:
Wrong implementation. Language must be extracted from a header in a middleware and be injected here

Reply:
src/app/middleware/middlewares/parse-localization.middleware.ts